python/sglang/srt/custom_op.py

```diff
@@ -36,10 +37,13 @@ def forward_cpu(self, *args, **kwargs):
         return self.forward_native(*args, **kwargs)
 
     def dispatch_forward(self):
+        from sglang.srt.managers.schedule_batch import global_server_args_dict
         if _is_cuda:
             return self.forward_cuda
         elif _is_hip:
             return self.forward_hip
+        elif global_server_args_dict["device"] == "cpu":
```

Comment:
can we do something like:
```
_is_cpu = is_cpu()
```
at the front of this file, like `_is_cuda` and `_is_hip` and put `is_cpu()` in `cpu_util.py`